src/prog_models/utils/containers.py

```diff
@@ -175,9 +196,11 @@ def __repr__(self) -> str:
 
         returns: a string of dictionaries containing all the keys and associated matrix values
         """
-        if len(self.matrix) > 0 and len(self.matrix[0]) == 1:   #the matrix has rows and the first row/list has one value in it
+        warn('DictLikeMatrix will be deprecated after version 1.5 of ProgPy and replaced by Pandas.', DeprecationWarning, stacklevel=2)
```

Comment:
```suggestion
```

I don't know if we want this message since users won't know they're using DictLikeMatrixWrapper- they'll just continue to use *Container (which will then be a series)